pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -74,69 +77,61 @@ func NewAnacondaInstallerISOTree(buildPipeline *Build, anacondaPipeline *Anacond
 	return p
 }
 
-func (p *AnacondaInstallerISOTree) getOSTreeCommitSources() []ostree.SourceSpec {
-	if p.OSTreeCommitSource == nil {
+func (p *AnacondaInstallerISOTree) getOSTreeCommits() []ostree.CommitSpec {
+	if p.commitSpec == nil {
 		return nil
 	}
+	return []ostree.CommitSpec{*p.commitSpec}
+}
 
-	return []ostree.SourceSpec{
-		*p.OSTreeCommitSource,
+func (p *AnacondaInstallerISOTree) getContainerSpecs() []container.Spec {
+	if p.containerSpec == nil {
+		return []container.Spec{}
 	}
+	return []container.Spec{*p.containerSpec}
 }
 
-func (p *AnacondaInstallerISOTree) getOSTreeCommits() []ostree.CommitSpec {
-	if p.ostreeCommitSpec == nil {
-		return nil
+func (p *AnacondaInstallerISOTree) getContainerSources() []container.SourceSpec {
+	if p.ContainerSource == nil {
+		return []container.SourceSpec{}
+	}
+	return []container.SourceSpec{
+		*p.ContainerSource,
 	}
-	return []ostree.CommitSpec{*p.ostreeCommitSpec}
 }
 
 func (p *AnacondaInstallerISOTree) getBuildPackages(_ Distro) []string {
 	packages := []string{
 		"squashfs-tools",
 	}
 
-	if p.OSTreeCommitSource != nil {
+	if p.CommitSource != nil {
 		packages = append(packages, "rpm-ostree")
 	}
 
+	if p.ContainerSource != nil {
+		packages = append(packages, "skopeo")
+	}
+
 	if p.OSPipeline != nil {
 		packages = append(packages, "tar")
 	}
 
 	return packages
 }
 
-func (p *AnacondaInstallerISOTree) serializeStart(_ []rpmmd.PackageSpec, _ []container.Spec, commits []ostree.CommitSpec) {
-	if len(commits) == 0 {
-		// nothing to do
-		return
-	}
-
-	if len(commits) > 1 {
-		panic("pipeline supports at most one ostree commit")
+func (p *AnacondaInstallerISOTree) serializeStart(_ []rpmmd.PackageSpec, containers []container.Spec, commits []ostree.CommitSpec) {
+	if p.commitSpec != nil || p.containerSpec != nil {
+		panic("double call to serializeStart()")
 	}
-
-	p.ostreeCommitSpec = &commits[0]
```

Comment:
Why was this removed and why isn't the container source set as well (when it's defined)?  Does the pipeline work without setting the specs?

Reply:
I quickly generated the manifests and yeah the ostree commit pipelines for edge installers were dropped.
You can also see that the ISOs were all rebuilt in this PR [1], which means the manifests changed.  None of the existing images should be changing here.

[1] https://gitlab.com/redhat/services/products/image-builder/ci/images/-/jobs/5794994872

Reply:
You can very quickly inspect the manifest changes without resolving content (it generates fake content IDs) to compare:
```
go run ./cmd/gen-manifests -output manifests -packages=false -commits=false -containers=false
```